server/models/contact.coffee

```diff
@@ -28,6 +28,14 @@ module.exports = class Contact extends cozydb.CozyModel
         # vCard Name = splitted
         # (Familly;Given;Middle;Prefix;Suffix)
         n             : String
+        org           : String
+        title         : String
+        department    : String
+        bday          : String
+        nickname      : String
```

Comment:
Currently, the attempt is to make it work with major vendors, not to match the RFC. If you want to help me do it. But there is already so much work, that I'm not patient enough to deal with that cardinality.